swagger_documentation/index.html

```diff
@@ -2024,10 +2024,11 @@ <h2 id="upload-bash-of-script">Upload Bash Script</h2>
 # exit script if any command fails
 set -e
 
-#modify ACCESS_TOKEN, FILE_NAME and FILE_PATH according to your needs
+#modify ACCESS_TOKEN, FILE_NAME, FILE_PATH and ENV according to your needs
 ACCESS_TOKEN='insert access token here'
 FILE_NAME='test.txt'
 FILE_PATH='/path/to/your/file/'$FILE_NAME
+ENV='chosen environment' #Two possible values: "figsh" (for staging/testing environment) and "figshare" (for live environment)
```

Comment:
Just add a variable for specifying the domain, like it's done in: https://docs.figshare.com/#upload_files_example_upload_on_figshare